pkg/model/components/kindnet.go

```diff
@@ -40,19 +40,27 @@ func (b *KindnetOptionsBuilder) BuildOptions(o *kops.Cluster) error {
 		c.Version = "v1.8.0"
 	}
 
-	// Kindnet should masquerade well known ranges if kops is not doing it
-	if c.Masquerade == nil {
-		c.Masquerade = &kops.KindnetMasqueradeSpec{
-			Enabled: fi.PtrTo(true),
+	if clusterSpec.IsIPv6Only() {
```

Comment:
We need to disable NAT64 in Kindnet If is AWS and if is IPv6 only

Reply:
... or disable NAT64 gateway and allow IPv4 connectivity from the instance , I just throw the options, no strong opinion